config/settings/development.yml

```diff
@@ -0,0 +1,11 @@
+:ems:
+  :ems_workflows:
+    :runner_options:
+      :docker:
+        :network: host
```

Comment:
Having a development.yml meant I could move the `options["network"] = "host" if Rails.env.development?` out of the engine :tada:

Reply:
huh. I never saw those implications. that is very cool